scripts/releases.py

```diff
@@ -73,14 +73,17 @@ def main():
     parser.add_argument(
         "--release-date", default=None, help="Date of the release")
     parser.add_argument(
-        "--db-size", default=None, help="Size of the database for the release in GB")
+        "--db-size", default=None,
+        help="Size of the database for the release in GB")
     parser.add_argument("--format", default="tsv", help=(
         "Output format, includes 'string', 'csv', 'tsv' (default), and "
         "'json'. "
-        "'tsv' can be appended to the IDR studies.csv file with no further "
-        "processing. "
         "All other formats include headers and totals. "
         "'string' is the most human-readable (fixed width columns)."
```

Comment:
```suggestion
        "'string' is the most human-readable (fixed width columns). "
```

Reply:
space is necessary to have the correct spacing for the next sentence
![Screen Shot 2020-09-04 at 15 20 54](https://user-images.githubusercontent.com/2478303/92249583-3f393000-eec2-11ea-9012-d5da2f993aa3.png)